docker/conf/traefik.toml

```diff
@@ -159,5 +151,16 @@ address = ":8080"
 # Optional
 # Default: true
 #
-# exposedbydefault = true
+exposedByDefault = false
```

Comment:
I'm reverting this back to `exposedByDefault` to `true` as it will break local server extensions if it does not add `traefik.enable=true` in their config. It could affect local-server extensions. This is the default behaviour before the upgrade.

I decided to add this thinking of making services exposed in Traefik more controlled.

Reverting.